app/renderer/ui/components/query/radonQuery/index.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from "react"
+import { RADRetrieveArgs, CreateDataRequestParams as RadRequest, RADAggregateArgs, RADConsensusArgs } from "../../../../../common/runtimeTypes/wallet"
+const style = require("./style.scss")
+export interface RadonQueryState {
+  retrieve: Array<RADRetrieveArgs>,
+  aggregate: RADAggregateArgs,
+  consensus: RADConsensusArgs,
+}
+
+export interface RadonQueryProps {
+  className?: string,
+  style?: string,
+  request: RadRequest,
+  // script?: RadonScript,
+  // operator?: RadonOp
+}
+
+/**
+ * RadonQuery UI component
+ *
+ * @export
+ * @class RadonQuery
+ * @extends {React.Component<RadonQuery>}
+ */
+
```

Comment:
Add a comment explaining that this component has to render script components instead of rendering the information. 
```suggestion
// TODO: Update this component to use radon script component that has to be implemented in #581 
```